libyui-rake.gemspec

```diff
@@ -41,6 +41,6 @@ end
   spec.require_path = "lib"
 
   # dependencies
-  spec.add_runtime_dependency("rake", "> 10.0", "< 12")
+  spec.add_runtime_dependency("rake", "> 10.0", "< 999")
```

Comment:
just remove the third argument?

Reply:
then `gem build` would say
"WARNING:  open-ended dependency on rake (> 10.0) is not recommended"